doc-content/enterprise-only/processes/filtering-process-instances-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='filtering-process-instances-proc']
+= Filtering process instances using basic or advanced filters
+
+{Central} now provides you with basic and advanced filters to help you filter and search through running process instances. You can filter processes by attributes such as *State*, *Errors*, *Filter By*, *Name*, *Start Date*, and *Last update*. You can also create custom filters using the *Advanced Filters* option. The newly created custom filter is added to the *Saved Filters* pane, which is accessible by clicking on the star icon on the left of the *Manage Process Instances* page.
+
+[float]
+== Filtering process instances using basic filters
+
+.Procedure
+. Go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the filter icon on the left of the page to expand the *Filters* pane.
++
+This pane lists the following process attributes which you can use to filter process instances:
++
+* *State*: Filter process instances based on their state (*Active*, *Aborted*, *Completed*, *Pending*, and *Suspended*).
+* *Errors*: Filter process instances that contain at least one or no errors.
+* *Filter By*: Filter process instances based on *Id*, *Initiator*, *Correlation Key*, or *Description* attribute.
++
+... Select the required attribute.
+... Enter the search query in the text field below.
+... Click *Apply*.
++
+
+* *Name*: Filter process instances based on process definition name.
+* *Start Date*: Filter process instances based on their creation date.
+* *Last update*: Filter process instances based on their last modified date.
+
+You can also use the *Advanced Filters* option to create custom filters in {CENTRAL}.
+
+[float]
+== Filtering process instances using advanced filters
+
+.Procedure
+. Go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click on *Advanced Filters* icon.
+. In the *Advanced Filters* pane, enter the name and description of the filter, and click *Add New*.
+. Select an attribute from the *Select column* drop-down list, for example, *processName*. The content of the drop-down changes to *processName != value1*.
+. Click the drop-down again and choose the required logical query. For the *processName* attribute, choose *equals to*.
+. Change the value of the text field to the name of the process you want to filter.
++
+[NOTE]
+====
+The name must match the value defined in the business process of the project.
```

Comment:
Would change here `The name` to `The processName`

Reply:
Changed as suggested.